_posts/en/newsletters/2023-07-19-newsletter.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'Bitcoin Optech Newsletter #260'
+permalink: /en/newsletters/2023/07/19/
+name: 2023-07-19-newsletter
+slug: 2023-07-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+_No significant news was found this week on the Bitcoin-Dev or
+Lightning-Dev mailing lists._
+
+## Waiting for confirmation #10: Get Involved
+
+_The final entry in our limited weekly [series][policy series] about
+transaction relay, mempool inclusion, and mining transaction
+selection---including why Bitcoin Core has a more restrictive policy
+than allowed by consensus and how wallets can use that policy most
+effectively._
+
+{% include specials/policy/en/10-get-involved.md %}
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Wallet 10101 beta testing DLCs on Lightning:**
+  10101 announced a [wallet][10101 github] built with LDK and BDK that allows users to trade
+  derivatives non-custodially using [DLCs][topic dlc] over [Lightning][10101 blog2]. The DLCs rely on
+  oracles that use [adaptor signatures][topic adaptor signatures] for price
+  [attestation][10101 blog1].
```

Comment:
@bitschmidty I'm uncomfortable with this description.  The way [they describe](https://10101.finance/blog/dlc-to-lightning-part-2/) their Lightning-based DLCs, they're just extending an LN commitment transaction with an extra commitment to a DLC.  That operation is strictly P2P between two counterparties; it's not routable like an LN payment.

To the best of my knowledge, it's not practical to do long-duration DLCs over LN because that would require long-duration HTLCs and, even if you found nodes willing to support HTLCs that wouldn't settle for weeks or months, the contracting parties would need to compensate the forwarding for their locked-up capital.  For example, if there are 10 hops, and the risk-free interest rate is 2%, and you want to enter a 3-month contract, the minimum rational forwarding fee is going to be 6.67% of the contract amount.  At that cost, for even a contract as small as $100, you're probably better off paying normal onchain transactions fees for the DLC (one transaction to open, one to settle, both of them small if it ends in a mutual close).

Maybe instead of saying "using DLCs over Lightning" we could say "using DLCs in an offchain contract that can also be used to send, receive, and forward LN payments".  Also maybe change the title to "Wallet 10101 beta testing pooling funds between LN and DLCs".

Reply:
Dave thanks for the feedback, I had a message to this team earlier in the week, but didn’t get a response. I largely took your suggestions.